src/api-server-selector.js

```diff
@@ -314,15 +325,34 @@ export class ApiServerSelector extends EventsTargetMixin(AmfHelperMixin(LitEleme
     this.uri = uri;
   }
 
+  /**
+   * Retrieves custom base uris elements assigned to the
+   * custom-base-uri slot
+   *
+   * @return {Array} Elements assigned to custom-base-uri slot
+   */
+  _getExtraServers() {
+    const slot = this.shadowRoot.querySelector('slot[name="custom-base-uri"]');
+    return slot ? Array.from(slot.assignedElements()) : [];
```

Comment:
You said it yourself, the `assignedElements()` returns an array so there's no point of having this :)
This would be true for the previous `assignedNodes()` function.

Reply:
Shoot, I missed that one